misc/assets/inforequests.svg

```diff
@@ -0,0 +1,248 @@
+<svg version="1.1" viewBox="0 0 1000 1350" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .external{
+        stroke-dasharray: 3pt;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .note {
+        font-size: 9pt;
+        text-anchor: middle;
+        dominant-baseline: ideographic;
+      }
+      .left-rotation{
```

Comment:
@martinmacko47 nazval som to "left-rotation", skusal som vsetky moznosti pre `dominant-baseline`, najviac sa mi hodilo `ideographics`, takze som to uz explicitne nedopisolval kedze uz to je v classe `note`

Reply:
ok